nx-cloud.example.env

```diff
@@ -0,0 +1,4 @@
+# Create a file named nx-cloud.env in the root of the repo and obtain the contents from the Platform Engineering Shared 1Password vault
```

Comment:
Under what circumstance would I want to do this?

Reply:
if you want to save cache into the cloud from local

Reply:
I'll make that clear.

Reply:
What would be the use case for that?

Reply:
engineer 1 has a branch and build locally in development -> cache is stored in the cloud.
engineer 2 checkout the branch and build it locally -> cache will be from the cloud.

Reply:
Is the intent of engineer 1 to make engineer 2's builds faster? So for example, I would write to nx-cloud so that my team mate can build the same branch faster?

Reply:
yes. that's the idea.